src/scss/grommet-core/_objects.pulse.scss

```diff
@@ -0,0 +1,77 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+$base-size: 40px;
```

Comment:
can we make this a 24px multiple following the inuitspacing units?

maybe: `$base-size double($inuit-base-spacing-unit);`